src/components/AppContainer.jsx

```diff
@@ -0,0 +1,18 @@
+import React, { Component, Fragment } from "react";
+import HeaderApp from "./Header";
+import FooterApp from "./Footer";
+import "../styles/AppContainer.css";
+
+export default class AppContainer extends Component {
```

Comment:
Do not use default import.

Use `export class AppContainer extends Component {` instead.
And that import it by using `import { AppContainer } from '<path>'`

https://blog.neufund.org/why-we-have-banned-default-exports-and-you-should-do-the-same-d51fdc2cf2ad